hotel/src/main/java/com/andersenhotels/model/storage/db_storage/LiquibaseRunner.java

```diff
@@ -0,0 +1,33 @@
+package com.andersenhotels.model.storage.db_storage;
+
+import com.andersenhotels.model.config.ConfigManager;
+import liquibase.Contexts;
+import liquibase.Liquibase;
+import liquibase.database.Database;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.resource.ClassLoaderResourceAccessor;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+
+public class LiquibaseRunner {
+
+    public static void runLiquibaseMigrations() {
+        try (Connection connection = DriverManager.getConnection(
+                ConfigManager.getDatabaseUrl(),
+                ConfigManager.getDatabaseUsername(),
+                ConfigManager.getDatabasePassword())) {
+
+            Database database = new liquibase.database.core.MySQLDatabase();
+            database.setConnection(new JdbcConnection(connection));
+
+            try (Liquibase liquibase = new Liquibase(ConfigManager.getLiquibaseChangeLogFile(), new ClassLoaderResourceAccessor(), database)) {
+                liquibase.update(new Contexts());
+            }
+            System.out.println("Liquibase migrations executed successfully.");
```

Comment:
As I can see you try to use logger in some laces, so try to use it instead of this. This part could also be a log message
And check in other places as well, please